api/ipfsproxy/config.go

```diff
@@ -250,8 +258,14 @@ func (cfg *Config) toJSONConfig() (jcfg *jsonConfig, err error) {
 	jcfg.NodeHTTPS = cfg.NodeHTTPS
 
 	jcfg.ExtractHeadersExtra = cfg.ExtractHeadersExtra
-	jcfg.ExtractHeadersPath = cfg.ExtractHeadersPath
-	if !(cfg.ExtractHeadersTTL > 0) {
+	if cfg.ExtractHeadersPath == DefaultExtractHeadersPath {
+		jcfg.ExtractHeadersPath = ""
+	} else {
+		jcfg.ExtractHeadersPath = cfg.ExtractHeadersPath
+	}
+
+	if cfg.ExtractHeadersTTL == DefaultExtractHeadersTTL {
+		jcfg.ExtractHeadersTTL = ""
 		return
```

Comment:
Same as before. Keep consistency.

`if ttl := cfg.ExtractHeadersTTL; ttl != DefaultExtractHeadersTTL { jcfg.ExtractHeadersTTL = ttl.String() }`

do not return